### PR TITLE
First version final modification

### PR DESCRIPTION
**Pull request by gdozot2**

### CODE REVIEW

**Comment by gdozot2** [comment]
For display purpose, I must rework all the json files. The change is related to the url of the image. Previously, I used square and now medium size.

The changed code is at the end of the the change tab.

File: src/components/Canvas/ImageElement.vue [hunk 1]
```diff
@@ -59,11 +62,14 @@ import {
 export default {
   watch: {
     imageId: function(newVal) {
-      this.shouldRunAnimation = false;
+      clearTimeout(this.timeoutEvent);
+      this.shouldDisapear = false;
```
Comment (slecorne) [inline src/components/Canvas/ImageElement.vue]: disappear has 2 p

File: src/components/Canvas/css/relatedImageAnimation.css [hunk 2]
```diff
@@ -11,6 +11,20 @@
   }
 }
 
+
+.relatedImageDisapearBase {
```
Comment (slecorne) [inline src/components/Canvas/css/relatedImageAnimation.css]: disappear

File: src/components/Canvas/service/image_management_service.js [hunk 3]
```diff
@@ -1,19 +1,23 @@
 const getFactor = (height, width) => {
   switch (true) {
+    case width >= 1600 && height >= 1550:
+      return {sizeFactor: 9, moveWidthFactor: 2.8, moveTopHeightFactor: 1.2, moveBottomHeightFactor: 1.7, moveCentralHeightFactor: 15};
```
Comment (slecorne) [inline src/components/Canvas/service/image_management_service.js]: can you add a comment on top what means `sizeFactor ` `moveWidthFactor` `moveTopHeightFactor` `moveBottomHeightFactor` `moveCentralHeightFactor`

**Review by slecorne — CHANGES_REQUESTED** [review]

File: src/components/Canvas/ImageElement.vue (hunk 1 above)
Comment (gdozot2) [reply src/components/Canvas/ImageElement.vue]: Remove, this attribute never used in the current implementation

File: src/components/Canvas/css/relatedImageAnimation.css (hunk 2 above)
Comment (gdozot2) [reply src/components/Canvas/css/relatedImageAnimation.css]: Done

File: src/components/Canvas/service/image_management_service.js (hunk 3 above)
Comment (gdozot2) [reply src/components/Canvas/service/image_management_service.js:14]: Done

**Review by slecorne — APPROVED** [review]

**Pull request merged**